neuralprophet/plot_model_parameters_plotly.py

```diff
@@ -99,9 +101,9 @@ def plot_trend_change(m, quantile, plot_name="Trend Change", df_name="__df__"):
     start = data_params["ds"].shift
     scale = data_params["ds"].scale
     time_span_seconds = scale.total_seconds()
-    cp_t = []
+    cp_t = np.array([])
```

Comment:
Good that we use numpy here. We should in general be strict in. using numpy arrays instead of lists.